config/jobs/kubernetes/sig-cloud-provider/azure/release-1.24.yaml

```diff
@@ -14,6 +14,7 @@ presubmits:
       preset-kind-volume-mounts: "true"
       preset-azure-cred-only: "true"
       preset-azure-anonymous-pull: "true"
+      preset-azure-capz-sa-cred: "true"
```

Comment:
these jobs should be generated from https://github.com/kubernetes/test-infra/blob/0cc6824beb2891fd5c2d79df3fd1684176396b53/config/jobs/kubernetes/sig-cloud-provider/azure/generate.sh, see the readme

Reply:
Fixed